src/pages/authorization/agent-authorization/index.mdx

```diff
@@ -43,9 +43,9 @@ To sign a request with a Personal Access Token, you need to:
 
 ## Step 1: Generate a PAT
 
-Start in <a href="https://platform.text.com/console/" target="_blank">Developer Console</a>. Then, go to **Tools -> Personal Access Tokens** and create a new token together with necessary [scopes](/authorization/scopes/). You won't be able to change those scopes once you create the token.
+Start in <a href="https://platform.text.com/console/" target="_blank">Developer Console</a>. Then, go to **Settings > Authorization > Personal Access Tokens** and create a new token together with necessary [scopes](/authorization/scopes/). You won't be able to change those scopes once you create the token.
 
-<img alt="LiveChat Personal Access Tokens" src="/images/authorization/livechat-personal-access-tokens-dark-2x.png" width="800px" height="445px"/>
+<img alt="LiveChat Personal Access Tokens" src="/images/authorization/livechat-personal-access-token.png" width="2142px" height="1262px"/>
```

Comment:
The image doesn't load — I think you forgot to add the file :D

Reply:
You're right. I haven't commited :P